backend/scripts/cases_load.py

```diff
@@ -0,0 +1,65 @@
+import csv  # https://docs.python.org/3/library/csv.html
+
+# https://django-extensions.readthedocs.io/en/latest/runscript.html
+
+# python3 manage.py runscript many_load
+import pandas as pd
+import pycountry
+from cases.models import Cases
+
+def run():
+
+    url_cases = 'https://raw.githubusercontent.com/alan-turing-institute/CounterfactualCovid19-inputs/develop/Data/Formatted/Cases_deaths_data_europe.csv'
+    df_cases = pd.read_csv(url_cases)
+
+    countries = df_cases.Country.unique()
+    date = '2020-06-23'
+
+    Cases.objects.all().delete()
+
+    for country in countries:
+
+        ts_data_country_daily, ts_data_country_cum = get_natural_history_data(country, df_cases)
+
+        total_cases = ts_data_country_daily[ts_data_country_daily.index < date].sum()
```

Comment:
I think we can replace this with native Pandas constructs without needing a separate function. Something like:

```
df_cases = pd.read_csv(url_cases, parse_dates=["Date"])
df_cases_filtered = df_cases[df_cases['Country'] == country][df_cases['Date'] < date]
```

Taking `Albania` as an example, this gives us a dataframe like the one below, which I think we can pull the cases from directly.

```
     Country       Date  Cumulative_cases_beg  Daily_cases  Cumulative_cases_end  Cumulative_cases_beg_MA7  Daily_cases_MA7  Cumulative_cases_end_MA7  ...  Daily_deaths  Cumulative_deaths_end  Cumulative_deaths_beg_MA7  Daily_deaths_MA7  Cumulative_deaths_end_MA7      Date_0 Date_pop_pct    Date_max
0    Albania 2020-01-22                     0            0                     0                     0.000            0.000                     0.000  ...             0                      0                      0.000             0.000                      0.000  2020-03-09   2020-03-11  2020-11-15
1    Albania 2020-01-23                     0            0                     0                     0.000            0.000                     0.000  ...             0                      0                      0.000             0.000                      0.000  2020-03-09   2020-03-11  2020-11-15
2    Albania 2020-01-24                     0            0                     0                     0.000            0.000                     0.000  ...             0                      0                      0.000             0.000                      0.000  2020-03-09   2020-03-11  2020-11-15
3    Albania 2020-01-25                     0            0                     0                     0.000            0.000                     0.000  ...             0                      0                      0.000             0.000                      0.000  2020-03-09   2020-03-11  2020-11-15
4    Albania 2020-01-26                     0            0                     0                     0.000            0.000                     0.000  ...             0                      0                      0.000             0.000                      0.000  2020-03-09   2020-03-11  2020-11-15
..       ...        ...                   ...          ...                   ...                       ...              ...                       ...  ...           ...                    ...                        ...               ...                        ...         ...          ...         ...
148  Albania 2020-06-18                  1722           66                  1788                  1717.432           63.000                  1780.432  ...             1                     39                     38.728             1.143                     39.871  2020-03-09   2020-03-11  2020-11-15
149  Albania 2020-06-19                  1788           50                  1838                  1780.432           57.857                  1838.289  ...             3                     42                     39.871             1.143                     41.014  2020-03-09   2020-03-11  2020-11-15
150  Albania 2020-06-20                  1838           53                  1891                  1838.289           53.571                  1891.860  ...             1                     43                     41.014             1.143                     42.157  2020-03-09   2020-03-11  2020-11-15
151  Albania 2020-06-21                  1891           71                  1962                  1891.860           56.000                  1947.860  ...             1                     44                     42.157             1.286                     43.443  2020-03-09   2020-03-11  2020-11-15
152  Albania 2020-06-22                  1962           33                  1995                  1947.860           57.714                  2005.574  ...             0                     44                     43.443             1.429                     44.872  2020-03-09   2020-03-11  2020-11-15

[153 rows x 17 columns]
```

Reply:
I feel we might have to do this a lot of times when building the different data models in the future, that is why I made it into a function. Let's discuss this more in detail on Tuesday.